docs/commands/browser/restoreState.mdx

```diff
@@ -0,0 +1,70 @@
+import Admonition from "@theme/Admonition";
+
+# restoreState
+
+## Overview {#overview}
+
+Browser command that restores session state (cookies, local and session storages) from a file or variable.
+
+<Admonition type="warning">
+    If you are using the `webdriver` automation protocol, you have to provide `webSocketUrl: true`
+    in `desiredCapabilities` in config browser settings. For the `devtools` protocol, you don't need
+    additional settings.
+</Admonition>
+
+## Usage {#usage}
+
+You can restore state from file (using the `path` param) or from object (using the `data` param).
+But if you provide both, the file will have the highest priority.
+Also, you can provide `cookies`, `localStorage` and `sessionStorage` params to restore only what you need.
+Data for restore state you can get from the [saveState](../saveState) command.
```

Comment:
We need a warning block over here, that user must be on the page from which the cookies were saved. If the user's on `about:blank` or on the other site, the error will be thrown.

Reply:
FIxed